tests/test_image_manager.py

```diff
@@ -9,83 +10,116 @@
 
 @mock_s3
 class TestImageManager(unittest.TestCase):
-    bucket = 'jf-test555-bucket'
-    FILENAME_TO_UPLOAD = "tests/test2.jpg"
+    BUCKET = 'jf-test555-bucket'
+    VECTOR_TO_UPLOAD = "images/1.eps"
+    VECTOR_FILE_EXT = '.eps'
+    VECTOR_SUBFOLDER = 'vector'
+    ITEM_ID_1 = "1e852a2d-35c2-409e-ac86-38224f5ac2d7"
+    PNG_TO_UPLOAD = "images/test.png"
 
     def setUp(self):
         aws_connector = AwsConnector()
         self.s3_resource = aws_connector.get_s3_resource()
-        self.s3_resource.create_bucket(Bucket=self.bucket, CreateBucketConfiguration={
+        self.s3_resource.create_bucket(Bucket=self.BUCKET, CreateBucketConfiguration={
             'LocationConstraint': 'eu-west-2'},)
-        self.image_manager = ImageManager(self.s3_resource, self.bucket)
+        self.image_manager = ImageManager(self.s3_resource, self.BUCKET)
 
     def tearDown(self):
-        bucket = self.s3_resource.Bucket(self.bucket)
+        bucket = self.s3_resource.Bucket(self.BUCKET)
         for key in bucket.objects.all():
             key.delete()
         bucket.delete()
 
-    def test_add_image(self):
-        # arrange
-        id = helper.get_id()
-        vector_img = self.image_manager._get_img_path(self.FILENAME_TO_UPLOAD, id)
-        png_img = self.image_manager._get_img_path(self.FILENAME_TO_UPLOAD, id, False)
-
+    def test_upload_vector_file(self):
         # act
-        self.image_manager.add_image(self.FILENAME_TO_UPLOAD, id, True)
-        self.image_manager.add_image(self.FILENAME_TO_UPLOAD, id, False)
+        vector_path = self.image_manager.upload_vector_file(
+            self.VECTOR_TO_UPLOAD, self.ITEM_ID_1)
         images = tuple(
```

Comment:
This line is more likely a part of the assert than act.

Reply:
fixed